lib/main.js

```diff
@@ -29,70 +33,79 @@ module.exports = class Client {
         });
     }
 
-
     /**
      * Upload asset feed to asset server
      */
 
-    uploadFeed (files = [], options = {}) {
+    uploadFeed (files, options = {}) {
+        assert(files, `Expected 'files' to be an array, instead got ${files}`);
```

Comment:
How do we feel about this change? It seems to me that there should never be an occasion to call `uploadFeed` with no parameters as it would do nothing useful. In addition to accidentally pass in `null` or `undefined` constitutes a usage error and the user should be informed.